src/cbmc_starter_kit/setup.py

```diff
@@ -73,8 +73,12 @@ def main():
     source_root = repository.repository_root()
     if shutil.which("litani"):
         litani = Path("litani")
-    else:
+    elif repository.litani_root() is not None:
         litani = repository.litani_root() / "litani"
+    else:
+        logging.error("could not find litani root. Is litani installed?")
```

Comment:
This is great.  Please consider an error message like "Can't find litani: see installation instructions at https://github.com/awslabs/aws-build-accumulator/releases/latest"